lib/src/state_manager/rx/rx_obx.dart

```diff
@@ -71,3 +83,59 @@ class _ObxState extends State<Obx> {
   @override
   Widget build(BuildContext context) => notifyChilds;
 }
+
+/// Similar to Obx, but manages a local state.
+/// Pass the initial data in constructor.
+/// Useful for simple local states, like toggles, visibility, themes,
+/// button states, etc.
+///  Sample:
+///    ObxValue((data) => Switch(
```

Comment:
For what i understood, this example should have the casting `ObxValue<Controller>()`, is this correct?

Reply:
The generic type the part that fails when defined, my bad, i think i left a TODO for it. Im waiting @jonataslaw feedback, to avoid the exception... and I will make other samples soon.
This is not relates to a controller, uses a local state with a Rx value directly. For instance, to toggle a button for toggleVisibility in a textfield, changing a Theme, use an integer for tab navigation/bottom navigation... those type of simple local variables.

Reply:
Can this work with Something like VideoPlayerController, Say you would like to initialise a VideoPlayerController for Playback?